cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/AppFabricServerTest.java

```diff
@@ -58,4 +61,25 @@ public void startStopServer() throws Exception {
     TimeUnit.SECONDS.sleep(1);
     Assert.assertTrue(Iterables.isEmpty(discovered));
   }
+
+  @Test
+  public void testSSL() {
+    CConfiguration cConf = CConfiguration.create();
+    cConf.setBoolean(Constants.Security.AppFabric.SSL_ENABLED, true);
+    cConf.setInt(Constants.AppFabric.SERVER_SSL_PORT, 20443);
+    SConfiguration sConf = SConfiguration.create();
+    sConf.set(Constants.Security.AppFabric.SSL_KEYSTORE_PASSWORD, "secret");
+    final Injector injector = AppFabricTestHelper.getInjector(cConf, sConf, new AbstractModule() {
+      @Override
+      protected void configure() {
+        // no overrides
+      }
+    });
+
+    discoveryServiceClient = injector.getInstance(DiscoveryServiceClient.class);
+    AppFabricServer appFabricServer = injector.getInstance(AppFabricServer.class);
+    appFabricServer.startAndWait();
+    Assert.assertTrue(appFabricServer.isRunning());
+    appFabricServer.stopAndWait();
```

Comment:
what does this test verify? just that app fabric server can startup when ssl is enabled? is there a better way to verify that SSL is in fact enabled (like having a request fail)?